src/Config.ts

```diff
@@ -1,10 +1,9 @@
-
-const config = {
-    warframeRelicDropUrl: "https://drops.warframestat.us/data/relics.json",
-    warframeMarketItemUrl: "https://api.warframe.market/v1/items",
-    warframeItemsUrl: "https://api.warframestat.us/items/search/Relics?by=category",
-    warframeRelicDropInfoUrl: "https://drops.warframestat.us/data/info.json",
-    warframePatchlogsUrl: "https://raw.githubusercontent.com/WFCD/warframe-patchlogs/master/data/patchlogs.json"
+const Config = {
```

Comment:
```suggestion
export const Config = {
```

Reply:
uhhh, no? why export the same thing twice from the same file?